src/ucp/core/ucp_worker.c

```diff
@@ -2095,6 +2107,8 @@ void ucp_worker_create_vfs(ucp_context_h context, ucp_worker_h worker)
                             "num_all_eps");
     ucs_vfs_obj_add_ro_file(worker, ucp_worker_vfs_show_keepalive_ep_count,
                             NULL, 0, "keepalive/ep_count");
+    ucs_vfs_obj_add_ro_file(worker, ucp_worker_vfs_show_keepalive_round_count,
```

Comment:
can use `ucs_vfs_show_primitive` instead of declaring new showing function

Reply:
done